cmd/server-test/multi_analyzer_test.go

```diff
@@ -38,12 +38,22 @@ func (suite *MultiDummyIntegrationSuite) TearDownTest() {
 func (suite *MultiDummyIntegrationSuite) TestSuccessReview() {
 	suite.sendEvent(successJSON)
 
-	suite.GrepAll(suite.r, []string{
+	str := suite.GrepAll(suite.r, []string{
 		"processing pull request",
 		"posting analysis",
-		`"file":"another.go","line":3,"text":"This line exceeded`,
 		`status=success`,
 	})
+
+	expr1 := `level=warning msg="analyzer Dummy2 generated 2 duplicated comments".*` +
+		`{"analyzer-name":"Dummy1","file":"another\.go","text":"The file has increased in 4 lines\."}.*` +
+		`{"analyzer-name":"Dummy1","file":"another\.go","line":3,"text":"This line exceeded 120 chars\."}`
+	expr2 := `level=warning msg="analyzer Dummy1 generated 2 duplicated comments".*` +
+		`{"analyzer-name":"Dummy2","file":"another\.go","text":"The file has increased in 4 lines\."}.*` +
+		`{"analyzer-name":"Dummy2","file":"another\.go","line":3,"text":"This line exceeded 120 chars\."}`
+	expr := `(?s)(` + expr1 + `|` + expr2 + `)`
+
+	found := suite.EgrepWholeFromString(str, expr)
+	suite.Require().True(found)
```

Comment:
Now the test check that the server warns about the duplicated comments, but we are not testing that any comment is actually posted, right?

Reply:
My bad, I thought this was all a warning message
```go
	expr1 := `level=warning msg="analyzer Dummy2 generated 2 duplicated comments".*` +
		`{"analyzer-name":"Dummy1","file":"another\.go","text":"The file has increased in 4 lines\."}.*` +
		`{"analyzer-name":"Dummy1","file":"another\.go","line":3,"text":"This line exceeded 120 chars\."}`

```
but it's actually the warning of duplicated and the 2 comments posted :+1: